ui/Makefile

```diff
@@ -54,22 +56,34 @@ GOBINDATA_SOURCES       := $(BOWER_COMPONENTS)/d3/d3.min.js \
                            $(BOWER_COMPONENTS)/mithriljs/mithril.js \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.min.css \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.min.js \
+                           $(BOWER_COMPONENTS)/moment/min/moment.min.js \
+                           $(BOWER_COMPONENTS)/moment-timezone/builds/moment-timezone-with-data.min.js \
                            $(GOBINDATA_FONTS)
 GOBINDATA_DEBUG_SOURCES := $(BOWER_COMPONENTS)/d3/d3.js \
                            $(BOWER_COMPONENTS)/lodash/lodash.js \
                            $(BOWER_COMPONENTS)/mithriljs/mithril.js \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.css \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.js \
+                           $(BOWER_COMPONENTS)/moment/moment.js \
+                           $(BOWER_COMPONENTS)/moment-timezone/builds/moment-timezone-with-data.js \
```

Comment:
Currently I just use it for UTC. That said, I can also probably strip it out in favor of javascript Date's UTC functions. Furthermore, we can remove it if the conversion is done on the backend.